lib/insert/insert_refunds.rb

```diff
@@ -22,9 +22,13 @@ def self.modern_refund(charge,h)
       nonprofit_id: {required: true, is_integer: true},
       supporter_id: {required: true, is_integer: true}
     })
+    if h['amount'].to_i != nil
+      ## TODO DO NOT MERGE AS IS
+      h['amount'] = h['amount'].to_i
+    end
```

Comment:
I think this means we can't merge it yet haha

Reply:
But why didn't I want to merge as-is?

Reply:
I've removed the warning, I don't see any reason why it's necessary.